include/turtle/detail/type_name.hpp

```diff
@@ -24,7 +28,7 @@
 #include <cstdlib>
 #endif
 
-#define MOCK_TYPE_NAME( t ) mock::detail::type_name( BOOST_SP_TYPEID(t) )
+#define MOCK_TYPE_NAME( t ) mock::detail::type_name( BOOST_CORE_TYPEID(t) )
```

Comment:
Doesn’t this depend on which include was used?
I think this should be wrapped in a macro such as `MOCK_TYPEID`?

Reply:
For 1.66 the definition was `#define BOOST_SP_TYPEID(T) BOOST_CORE_TYPEID(T)` - thus it should not make any difference. Can not say for older versions though.

Reply:
Just took a quick look at 1.54 - u r right. Will change.